### ISSUE TITLE
More Descriptive Close Modal Button

### ISSUE THREAD

**Issue by jpdevries** [Sterc/mediamanager#46]
`aria-label` is 👍 

Normally I'd be fine leaving it at that, but until #45 is resolved, I think it should be more descriptive so it isn't confused with stuff that should be hidden from the a11y tree but isn't.

![](http://j4p.us/303f3F2o2N1J/Screen%20Shot%202017-07-05%20at%202.27.05%20PM.png)
